notebook/i18n/fr_FR/LC_MESSAGES/nbui.po

```diff
@@ -0,0 +1,746 @@
+msgstr "Rechercher et Remplacer"
+
+#: notebook/templates/notebook.html:152
+msgid "Cut Cell Attachments"
+msgstr "Couper les Pièces-Jointes de la Cellule"
+
+#: notebook/templates/notebook.html:153
+msgid "Copy Cell Attachments"
+msgstr "Copier les Pièces-jointes de la Cellule"
+
+#: notebook/templates/notebook.html:154
+msgid "Paste Cell Attachments"
+msgstr "Coller les Pièces-jointes de la Cellule"
+
+#: notebook/templates/notebook.html:156
+msgid "Insert Image"
+msgstr "Insérer une Image"
+
+#: notebook/templates/notebook.html:166
+msgid "Show/Hide the action icons (below menu bar)"
+msgstr "Afficher/Masquer les icônes d'action (en-dessous de la barre de menu)"
+
+#: notebook/templates/notebook.html:167
+msgid "Toggle Toolbar"
+msgstr "Afficher/Masquer la Barre d'Outils"
+
+#: notebook/templates/notebook.html:170
+msgid "Show/Hide line numbers in cells"
+msgstr "Afficher/Masquer les numéros de ligne dans les cellules"
+
+#: notebook/templates/notebook.html:174
+msgid "Cell Toolbar"
+msgstr "Barre d'outil de cellule"
+
+#: notebook/templates/notebook.html:179
+msgid "Insert"
+msgstr "Insérer"
+
+#: notebook/templates/notebook.html:182
+msgid "Insert an empty Code cell above the currently active cell"
+msgstr "Insérer une cellule de Code vide au-dessus de la cellule active"
+
+#: notebook/templates/notebook.html:183
+msgid "Insert Cell Above"
+msgstr "Insérer une Cellule au-Dessus"
+
+#: notebook/templates/notebook.html:185
+msgid "Insert an empty Code cell below the currently active cell"
+msgstr "Insérer une cellule de Code vide en-dessous de la cellule active"
+
+#: notebook/templates/notebook.html:186
+msgid "Insert Cell Below"
+msgstr "Insérer une Cellule en-Dessous"
+
+#: notebook/templates/notebook.html:189
+msgid "Cell"
+msgstr "Cellule"
+
+#: notebook/templates/notebook.html:191
+msgid "Run this cell, and move cursor to the next one"
+msgstr "Exécuter cette cellule, et déplacer le curseur à la suivante"
+
+#: notebook/templates/notebook.html:192
+msgid "Run Cells"
+msgstr "Exécuter les Cellules"
+
+#: notebook/templates/notebook.html:193
+msgid "Run this cell, select below"
+msgstr "Exécuter cette cellule et sélectionner la suivante"
+
+#: notebook/templates/notebook.html:194
+msgid "Run Cells and Select Below"
+msgstr "Exécuter les cellules et sélectionner celle en-dessous"
+
+#: notebook/templates/notebook.html:195
+msgid "Run this cell, insert below"
+msgstr "Exécuter la cellule et insérer après"
+
+#: notebook/templates/notebook.html:196
+msgid "Run Cells and Insert Below"
+msgstr "Exécuter les Cellules et insérer Après"
+
+#: notebook/templates/notebook.html:197
+msgid "Run all cells in the notebook"
+msgstr "Exécuter toutes les cellules du notebook"
+
+#: notebook/templates/notebook.html:198
+msgid "Run All"
+msgstr "Exécuter tout"
+
+#: notebook/templates/notebook.html:199
+msgid "Run all cells above (but not including) this cell"
+msgstr "Exécuter toutes les cellules au-dessus de celle-ci (non incluse)"
```

Comment:
(strictement)?

Reply:
It takes to be a matematician to understand that « strictement » use (IMHO), so I prefer my version, that have IMHO a more universal understanding.